README.md

```diff
@@ -13,20 +13,22 @@ development inside a Bazel workspace.
 ## Table of Contents
 
 <!-- MARKDOWN TOC: BEGIN -->
-* [Documentation](#documentation)
-* [Prerequisites](#prerequisites)
-  * [Mac OS](#mac-os)
-  * [Linux](#linux)
-* [Quickstart](#quickstart)
-  * [1\. Configure your workspace to use <a href="https://github\.com/cgrindel/swift\_bazel">swift\_bazel</a>\.](#1-configure-your-workspace-to-use-swift_bazel)
-  * [2\. Create a minimal Package\.swift file\.](#2-create-a-minimal-packageswift-file)
-  * [3\. Add Gazelle targets to BUILD\.bazel at the root of your workspace\.](#3-add-gazelle-targets-to-buildbazel-at-the-root-of-your-workspace)
-  * [4\. Resolve the external dependencies for your project\.](#4-resolve-the-external-dependencies-for-your-project)
-  * [5\. Create or update Bazel build files for your project\.](#5-create-or-update-bazel-build-files-for-your-project)
-  * [6\. Build and test your project\.](#6-build-and-test-your-project)
-  * [7\. Check in some generated files\.](#7-check-in-some-generated-files)
-* [Tips and Tricks](#tips-and-tricks)
-* [Future Work](#future-work)
+- [Gazelle Plugin for Swift and Swit Package Rules for Bazel](#gazelle-plugin-for-swift-and-swit-package-rules-for-bazel)
```

Comment:
We have [a utility and test](https://github.com/cgrindel/swift_bazel/blob/main/BUILD.bazel#L135) that generates the TOC for our markdown files. Did you see something that was not correct in the generated TOC?

Reply:
sorry, my VSCode plugin change TOC.